Author: <agx@sigxcpu.org>
Description: Intialize all fields in uinput_user_dev, otherwise strange things 
like dead keyboard, etc. might happen
--- a/mouseemu.c
+++ b/mouseemu.c
@@ -418,6 +418,7 @@
 	struct uinput_user_dev device;
 	int i;
 	
+	memset(&device, 0, sizeof(struct uinput_user_dev));
 	/*setup keyboard device */
 	
         if(ui_keyb_fd > 0) {
